### PR TITLE
Add datasets for transcript quantification and differential expression analysis

### PR DESCRIPTION
**Pull request by yuukiiwa**
Hi @cying111, I have updated the fastq/demultiplexed datasets and corresponding fasta and gtf files for testing. They are small and can be downloaded quickly. I have also updated the samplesheets with specific headers.
Here is the run:
Screen Shot 2020-08-07 at 2 14 33 AM

### CODE REVIEW

File: samplesheet_nobc_nodx.csv [hunk 1]
```diff
@@ -1,3 +1,7 @@
-sample,fastq,barcode,genome,transcriptome
-K562_RUN1_REP3,https://raw.githubusercontent.com/nf-core/test-datasets/nanoseq/fastq/demultiplexed/barcode01.fastq.gz,,https://raw.githubusercontent.com/nf-core/test-datasets/nanoseq/reference/hg19_KCMF1.fa,
-HEPG2_RUN3_REP5,https://raw.githubusercontent.com/nf-core/test-datasets/nanoseq/fastq/demultiplexed/barcode02.fastq.gz,,,
+sample,sample_path,barcode,genome,transcriptome,condition
```
Comment (drpatelh) [inline samplesheet_nobc_nodx.csv:1]:
```suggestion
sample,input_file,barcode,genome,transcriptome,condition
```

Comment (drpatelh) [reply samplesheet_nobc_nodx.csv:1]: Can we use `input_file` here instead @yuukiiwa ? More generically applicable.

Comment (yuukiiwa) [reply samplesheet_nobc_nodx.csv]: Thanks for the reminder! I have fixed them in all samplesheets.

**Comment by yuukiiwa** [comment]
fastq input run with the updated test dataset:
fastq_run

File: samplesheet_nobc_nodx_noaln.csv [hunk 2]
```diff
@@ -0,0 +1,2 @@
+sample,input_file,barcode,genome,transcriptome,condition
+MCF7_directcDNA_replicate1,	https://raw.githubusercontent.com/nf-core/test-datasets/nanoseq/bam/MCF7_directcDNA_replicate1.bam,,	https://raw.githubusercontent.com/nf-core/test-datasets/nanoseq/reference/chr22_19431000-19528000.fa,	https://raw.githubusercontent.com/nf-core/test-datasets/nanoseq/reference/chr22_19431000-19528000.gtf,A
```
Comment (drpatelh) [inline samplesheet_nobc_nodx_noaln.csv:2]:
```suggestion
MCF7_directcDNA_replicate1,https://raw.githubusercontent.com/nf-core/test-datasets/nanoseq/bam/MCF7_directcDNA_replicate1.bam,,https://raw.githubusercontent.com/nf-core/test-datasets/nanoseq/reference/chr22_19431000-19528000.fa,https://raw.githubusercontent.com/nf-core/test-datasets/nanoseq/reference/chr22_19431000-19528000.gtf,MCF7
```

File: samplesheet_nobc_nodx.csv [hunk 3]
```diff
@@ -1,3 +1,7 @@
-sample,fastq,barcode,genome,transcriptome
-K562_RUN1_REP3,https://raw.githubusercontent.com/nf-core/test-datasets/nanoseq/fastq/demultiplexed/barcode01.fastq.gz,,https://raw.githubusercontent.com/nf-core/test-datasets/nanoseq/reference/hg19_KCMF1.fa,
-HEPG2_RUN3_REP5,https://raw.githubusercontent.com/nf-core/test-datasets/nanoseq/fastq/demultiplexed/barcode02.fastq.gz,,,
+sample,input_file,barcode,genome,transcriptome,condition
+MCF7_directcDNA_replicate1,https://raw.githubusercontent.com/nf-core/test-datasets/nanoseq/fastq/MCF7_directcDNA_replicate1.fastq.gz,,https://raw.githubusercontent.com/nf-core/test-datasets/nanoseq/reference/chr22_19431000-19528000.fa,https://raw.githubusercontent.com/nf-core/test-datasets/nanoseq/reference/chr22_19431000-19528000.gtf,MCF7
+MCF7_directcDNA_replicate3,https://raw.githubusercontent.com/nf-core/test-datasets/nanoseq/fastq/MCF7_directcDNA_replicate3.fastq.gz,,,,MCF7
```
Comment (drpatelh) [inline samplesheet_nobc_nodx.csv:3]: You don't.need to specify the reference fasta and gtf for all entries?

Comment (yuukiiwa) [reply samplesheet_nobc_nodx.csv]: Thanks for the reminder! I have specified them.

**Pull request merged**